model/CharacterProperties.py

```diff
@@ -1,23 +1,73 @@
-from abc import ABC, ABCMeta, abstractmethod
-
-
-class CharacterProperties(ABC): pass
-
-
-class Flinkheit(CharacterProperties): pass
-class Schwerfaelligkeit(CharacterProperties): pass
-class Behaendigkeit(CharacterProperties): pass
-class Behaebigkeit(CharacterProperties): pass
-class Agilitaet(CharacterProperties): pass
-class Glueckspilz(CharacterProperties): pass
-class Pechvogel(CharacterProperties): pass
-class KlammeKlamotten(CharacterProperties): pass
-class KonstantKlammeKlamotten(CharacterProperties): pass
-class RobusterMagen(CharacterProperties): pass
-class Zaehigkeit(CharacterProperties): pass
-class Babysitter(CharacterProperties): pass
-class HoneyTrap(CharacterProperties): pass
-class BangAndBurn(CharacterProperties): pass
-class FlapsAndSeals(CharacterProperties): pass
-class Tradecraft(CharacterProperties): pass
-class Observation(CharacterProperties): pass
+from abc import ABC
+
+
+class CharacterProperties(ABC):
+    pass
+
+
+class Nimbleness(CharacterProperties):
+    pass
+
+
+class Sluggishness(CharacterProperties):
+    pass
+
+
+class Spryness(CharacterProperties):
+    pass
+
+
+class Stolidity(CharacterProperties):
+    pass
+
+
+class Agility(CharacterProperties):
+    pass
+
+
+class Jinx(CharacterProperties):
+    pass
+
+
+class LuckyDevil(CharacterProperties):
+    pass
+
+
+class ClammyClothes(CharacterProperties):
+    pass
+
+
+class ConstantClammyClothes(CharacterProperties):
+    pass
+
+
+class RobustStomach(CharacterProperties):
+    pass
+
+
+class Toughness(CharacterProperties):
+    pass
+
+
+class Babysitter(CharacterProperties):
+    pass
+
+
+class HoneyTrap(CharacterProperties):
+    pass
+
+
+class BangAndBurn(CharacterProperties):
+    pass
+
+
+class FlapsAndSeals(CharacterProperties):
+    pass
+
+
+class Tradecraft(CharacterProperties):
+    pass
+
+
+class Observation(CharacterProperties):
+    pass
```

Comment:
Es gibt enums in python. Hier beispielhaft an CharacterProperties (an anderen Stellen auch verwendbar)

```suggestion
from enum import Enum


class CharacterProperties(Enum):
    Nimbleness
    Sluggishness
    Spryness
    Stolidity
    Agility
    Jinx
    LuckyDevil
    ClammyClothes
    ConstantClammyClothes
    RobustStomach
    Toughness
    Babysitter
    HoneyTrap
    BangAndBurn
    FlapsAndSeals
    Tradecraft
    Observation
```

Reply:
Für Properties ist das Ok.